src/properties/property-effects.html

```diff
@@ -117,6 +160,82 @@
   }
 
   /**
+   * Runs "notify" effects for a set of changed properties.
+   *
+   * This method differs from the generic `runEffects` method in that it
+   * will dispatch path notification events in the case that the property
+   * changed was a path and the root property for that path didn't have a
+   * "notify" effect.  This is to maintain 1.0 behavior that did not require
+   * `notify: true` to ensure object sub-property notifications were
+   * sent.
+   *
+   * @param {Object} inst The instance with effects to run
+   * @param {Object} props Bag of current property changes
+   * @param {Object} oldProps Bag of previous values for changed properties
+   * @private
+   */
```

Comment:
Moved to `property-effects` from `batched-effects` and given a better name (was previously `notifyProperties`); simplified from previous implementation since re-entry stuff was removed, as it is called